### ISSUE TITLE
Question regarding Usage

### ISSUE THREAD

**Issue by mutiny1** [bntjah/lancache#78]
My current setup looks nothing like this iteration, I use pfSense to forward the DNS requests to the lancache IP(s).  Does this particular setup handle all the DNS traffic and internally forward to the vIP of the cache?  Or do I still use pfSense to forward to this box?

**Comment by bntjah** [comment]
Hi Mutiny1,

You can continue to use pfsense for your dns and redirect certain domains to the lancache ip's.
As pfsense is using unbound standard config you could copy the config from in here in to your advanced pfsense; there is a text entry field somewhere, just c/p it in there and should work...

Just don't forget to modify the ips.

**Comment by mutiny1** [comment]
Excellent!  Exactly what I did, took the unbound.conf and reformatted for structure and put into pfSense.
It runs great on 16.04 LTS.
Thanks for your excellent work, I did however run into issues with the script installer, it was minor but frustrating at first.  The /etc/network/interfaces starts off good, then half way it drops the last octet from the VIPs; then the last 4 VIPs are just missing the IP altogether.  Do you want me to run the script again and really document my experience (and fixes) and open another case?

**Comment by nexusofdoom** [comment]
how old is the installer ?
what line on the install script? 

thanks

**Comment by mutiny1** [comment]
Taken from : https://github.com/bntjah/lc-installer
1) git clone -b master --recursive http://github.com/bntjah/lc-installer/
2) cd lc-installer
3) chmod +x installer.sh
4) ./installer.sh
Didn't check the unbound.conf until this morning, but it looked similar to the /etc/network/interfaces file.
[unbound.conf.txt](https://github.com/bntjah/lancache/files/1583120/unbound.conf.txt)

**Comment by nexusofdoom** [comment]
it should pull from the hosts file once setup

**Comment by nexusofdoom** [comment]
lc-host-apple lancache-apple
lc-host-wargaming lancache-wargaming
lc-host-uplay lancache-uplay
lc-host-glyph lancache-glyph
lc-host-zenimax lancache-zenimax
lc-host-digitalextremes lancache-digitalextremes
lc-host-perlabyss lancache-pearlabyss

**Comment by mutiny1** [comment]
I figured that, but in checking the hostfile it appears correct.
[hosts.txt](https://github.com/bntjah/lancache/files/1583135/hosts.txt)

**Comment by mutiny1** [comment]
I should clarify something.  I did not edit the script in any way.  I ran the 5 steps listed and then fixed issues as they arose.  
1. networking is dead since the interface file is missing IP's  --easy fix (just filled in blanks as needed), 
2. lancache service does not come up clean (i think it comes up before all the VIPs OR before my /srv/lancache/data is mounted)
  systemctl status lancache.service  
[CacheLogs.txt](https://github.com/bntjah/lancache/files/1583185/CacheLogs.txt)
3. waiting 30 sec and manually starting  systemctl start lancache.service seems to be fine.
[CacheLogs_wait.txt](https://github.com/bntjah/lancache/files/1583189/CacheLogs_wait.txt)

If you think this is because I'm using Ubuntu Srv 16.04LTS and not Debian 8.7.x then don't waste time on this.
